GP02HYPM/GP02HYPM_D00007_ingest.csv

```diff
@@ -1,6 +1,6 @@
 parser,filename_mask,reference_designator,data_source,status,notes
 mi.dataset.driver.sio_eng.sio.sio_eng_sio_telemetered_driver,/omc_data/whoi/OMC/GP02HYPM/D00007/*status*.dat,GP02HYPM-RIM01-00-SIOENG000,telemetered,Available,
-mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_sio_telemetered_driver,/omc_data/whoi/OMC/GP02HYPM/D00007/*ctdmo*.dat,GP02HYPM-RIM01-02-CTDMOG039,telemetered,Available,
+#mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_sio_telemetered_driver,/omc_data/whoi/OMC/GP02HYPM/D00007/*ctdmo*.dat,GP02HYPM-RIM01-02-CTDMOG039,telemetered,Not Expected,Files contain no valid data

```

Comment:
Looks good.

Reply:
looks good.  Just wondered if the other sensors on the upper WFP should say 'Not Expected' instead of 'Available'?  They are commented out.